src/ResourceManagement/Graph.RBAC/PasswordCredentialImpl.cs

```diff
@@ -123,7 +123,7 @@ internal PasswordCredentialImpl(Models.PasswordCredential passwordCredential)
         internal PasswordCredentialImpl(string name, IHasCredential<T> parent)
             : base(name, new Models.PasswordCredential()
             {
-                CustomKeyIdentifier = System.Convert.ToBase64String(System.Text.Encoding.UTF8.GetBytes(name)),
+                CustomKeyIdentifier = System.Text.Encoding.UTF8.GetBytes(name),
```

Comment:
Why change the encoding?

Reply:
this generated field is changed from base64string to byte by swagger